dbt/dbt_project.yml

```diff
@@ -28,6 +28,8 @@ clean-targets:
 models:
   athena:
     +materialized: view
+    +write_compression: SNAPPY
+    +format: PARQUET
```

Comment:
Here's where we set `write_compression` and `format` for all tables. The equivalent of `external_location` will be set in `profiles.yml` below.